### PR TITLE
Add test case for parsing a real quote

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #200** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/200" target="_blank">Graphite</a> 
    * **PR #202** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/202" target="_blank">Graphite</a> 
      * **PR #203** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/203" target="_blank">Graphite</a> 
        * **PR #204** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/204" target="_blank">Graphite</a> 
          * **PR #205** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/205" target="_blank">Graphite</a> 
            * **PR #208** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/208" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/208?utm_source=stack-comment).

File: dcap/types/assets/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Test assets for `mc-sgx-dcap-types`
+
+* `hw_quote.dat` A hardware quote taken from a live system. See
```
[the remark this answers is not in the document]
Comment (awygle) [reply dcap/types/assets/tests/README.md:3]: I guess I weakly prefer `data` to `assets` for consistency? I don't really mind either though.

**Review by NotGyro — APPROVED** [review]
Looks good to me

File: dcap/types/assets/tests/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nick-mobilecoin) [reply dcap/types/assets/tests/README.md]: renamed to `data`

**Pull request merged**